dataset_generation/hi_wikisplit/Scraper.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import requests
+import os
+import sys
+import shutil
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+class Scraper:
+    """
+    Uses the wikimedia api to get the revision histories of the titles 
+    provided in the title dump file.
+    Format varies between wikipedia languages
+    """
+    def __init__(self, titles_file, url = "https://hi.wikipedia.org/w/api.php",
+                 titles_per_chunk = 1000, output_dir = './raw_data/',
```

Comment:
have we added the raw_data directory too in the pull request?

Reply:
scraped data gets written to the raw_data directory by default, unless specified otherwise. i have not included any of the output files, though. should i include the output files?

Reply:
if it is not much of data, then yes we should otherwise we can skip

Reply:
the whole of the raw dataset would run in hundreds of thousands of lines (since this isnt cleaned and checked yet), but i can add a few files with smaller amounts of data, if thats okay.

Reply:
oh...then no worries...dont add the data for now. we can add samples of data in the next code iteration and not right now!

Reply:
cool!